lib/tarantool_server.py

```diff
@@ -663,7 +663,7 @@ def __init__(self, _ini=None, test_suite=None):
         # or passed through execfile() for PythonTest
         self.current_test = None
         caller_globals = inspect.stack()[1][0].f_globals
-        if 'test_run_current_test' in caller_globals.keys():
+        if 'test_run_current_test' in list(caller_globals):
```

Comment:
That's a dictionary object (I verified on both Python 2 and Python 3). `if 'foo' in <dictionary>` is enough. There is no logic in transformation to a list, another dict, a set, a string, some custom class or any other structure to check whether a key is in a dictionary.

In fact, `caller_globals.keys()` works on both Python 2 and Python 3, so this change is just unnecessary.

See also my previous [comment](https://github.com/tarantool/test-run/pull/263#discussion_r575164506) on this topic. 'It works' can advocate so wide variety of meaningless solutions that I cannot consider it as a valuable argument.

Reply:
reverted change